_aggregations/bucket/significant-terms.md

```diff
@@ -9,63 +9,233 @@ redirect_from:
 
 # Significant terms aggregations
 
-The `significant_terms` aggregation lets you spot unusual or interesting term occurrences in a filtered subset relative to the rest of the data in an index.
+`significant_terms` helps you surface terms that are unusually frequent in a subset of documents (foreground set) compared to a broader reference set (background set). It’s the right choice when a plain `terms` aggregation shows you the *most common* values, but you want the *most over‑represented* values.
 
-A foreground set is the set of documents that you filter. A background set is a set of all documents in an index.
-The `significant_terms` aggregation examines all documents in the foreground set and finds a score for significant occurrences in contrast to the documents in the background set.
+- Foreground set: the documents matched by your query.
+- Background set: by default, all documents in the target indexes. You can narrow it with `background_filter`.
 
-In the sample web log data, each document has a field containing the `user-agent` of the visitor. This example searches for all requests from an iOS operating system. A regular `terms` aggregation on this foreground set returns Firefox because it has the most number of documents within this bucket. On the other hand, a `significant_terms` aggregation returns Internet Explorer (IE) because IE has a significantly higher appearance in the foreground set as compared to the background set.
+Each result bucket includes:
+
+- `key`: the term value.
+- `doc_count`: the number of foreground docs containing the term.
+- `bg_count`: the number of background docs containing the term.
+- `score`: how strongly the term stands out in the foreground relative to the background, see [Heuristics and scoring](#heuristics-and-scoring) for further details.
+
+If the aggregation returns no buckets, you likely didn’t filter the foreground, for example using `match_all`, or the foreground has the same distribution of terms as the background.
+{: .note}
+
+## Basic example: Find out what's distinctive about high‑value returns
+
+You can use the following query to ask, among orders that were *returned and cost over 500*, which `payment_method` values are unusually common compared to the whole index?
 
 ```json
-GET opensearch_dashboards_sample_data_logs/_search
+GET retail_orders/_search
```

Comment:
I suggest to also provide example indexing commands and the output of this search request, so user can easily follow.